src/dehydrated_devices.rs

```diff
@@ -21,6 +21,51 @@ impl From<dehydrated_devices::DehydratedDevices> for DehydratedDevices {
     }
 }
 
+/// Dehydrated device key
+#[wasm_bindgen]
+#[derive(Debug)]
+pub struct DehydratedDeviceKey {
+    pub(crate) inner: Uint8Array,
```

Comment:
Also: does it need to be `pub(crate)`, or could it be private?

Reply:
looks like I had a bad understanding of pub(crate). Thx